src/Trace/Integrations/Eloquent.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Trace\Integrations;
+
+/**
+ * This class handles instrumenting the Eloquent ORM queries using the stackdriver_trace extension.
+ *
+ * Example:
+ * ```
+ * use Google\Cloud\Trace\Integrations\Eloquent
+ *
+ * Eloquent::load();
+ */
+class Eloquent
+{
+    /**
+     * Static method to add instrumentation to the Symfony framework
+     */
+    public static function load()
+    {
+        if (!extension_loaded('stackdriver_trace')) {
+            return;
+        }
+
+        stackdriver_trace_method('Illuminate\Database\Eloquent\Builder', 'getModels', function ($scope, $columns) {
```

Comment:
It'll only crash if you call the method which invokes `Builder::class`. Just adding the `use` won't break anything. I may be misunderstanding the way this works, but isn't it up to the user to register these different trace handlers? So by default, none of these would be invoked unless a user knows the classes exist and want to trace them?